examples/llm_finetune.py

```diff
@@ -72,7 +72,7 @@ def train(db, model_identifier, model_name, output_dir):
         logging_steps=5,
         gradient_checkpointing=True,
         report_to=[],
-        # deepspeed=deepspped,
+        # deepspeed=deepspeed,
```

Comment:
comment?

Reply:
Yes, Local single GPU or mac operation does not require deepspeed.
In addition, deepspeed cannot be installed on mac unless we configure the remote ray_address to point to the ray cluster with gpu

Reply:
Keep this comment because I want to keep an option, I will add documentation later to explain how to configure the use of different scenarios

Reply:
oh ok